src/index.html

```diff
@@ -5,8 +5,454 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <div class="img">
+        <a
+          href="#home"
+          class="a-image"
+        >
+          <img
+            src="images/logo.png"
+            alt="moyo"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#APPLE"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#SAMSUNG"
+              class="is-normal"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#СМАРТФОНЫ И ТЕЛЕФОНЫ"
+              class="is-normal"
+            >
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#НОУТБУКИ И КОМПЬЮТЕРЫ"
+              class="is-normal"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#ГАДЖЕТЫ"
+              class="is-normal"
+            >
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#ПЛАНШЕТЫ"
+              class="is-normal"
+            >
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#ФОТО"
+              class="is-normal"
+            >
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#ВИДЕО"
+              class="is-normal"
+            >
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="block-product">
+      <div
+      data-qa="card"
+      class="block-product__screen screen"
+    >
```

Comment:
Just pay attention to class names
It's not block-product, will be better to name it `catalog`
And each card can be `catalog__card`